api/src/wfl/executor.clj

```diff
@@ -217,24 +232,27 @@
         (when-not (== (count records) (count workflows))
           (throw (ex-info "Allocated more records than created workflows"
                           {:submission submission :executor executor})))
+        (log/debug (format "%s Writing %s workflow uuids and statuses for submission %s..."
+                           (log-prefix executor) (count workflows) submission))
         (->> workflows
              (map #(update % :workflowEntity :entityName))
              (sort-by :workflowEntity)
              (map zip-record (sort-by :entity records))
              (write-workflow-statuses (utc-now)))))))
 
+(def ^:private final-statuses (util/to-quoted-comma-separated-list rawls/final-statuses))
```

Comment:
Calling this out, I used the Rawls final statuses instead of the Cromwell statuses since we're talking to Rawls to get our workflows.  They do match, but the active statuses differ between Rawls and Cromwell.